### DISCUSSION TITLE
Laravel InsideAuth

### DISCUSSION THREAD

**Discussion by nahime0** [HassanZahirnia/laravel-package-ocean#8] [Package Suggestions]
Laravel InsideAuth is a seamless side authentication solution for Laravel packages. It provides an easy and independent authentication mechanism, allowing developers to focus on building core features of their packages or applications without worrying about managing separate authentication systems.

[GitHub](https://github.com/illegalstudio/insideauth)

**Comment by HassanZahirnia** [comment]
Hey, thanks for your suggestion. I will hold onto adding your package for now since you marked it as not ready for production. Will recheck later!
